include/z64.h

```diff
@@ -1446,4 +1446,55 @@ typedef struct {
     /* 0x18 */ f32 yScale;
 } CfbInfo; // size = 0x1C
 
+typedef struct {
+    /* 0x00 */ u16 table[8*8];
+} JpegQuantizationTable; // size = 0x80
+
+typedef struct {
+    /* 0x00 */ u8 codeOffs[16];
+    /* 0x10 */ u16 codesA[16];
+    /* 0x30 */ u16 codesB[16];
+    /* 0x50 */ u8* symbols;
+} JpegHuffmanTable; // size = 0x54
+
+typedef struct {
+    /* 0x00 */ u32 unk_00;
+    /* 0x04 */ u32 unk_04;
+    /* 0x08 */ u32 unk_08;
+    /* 0x0C */ u32 qTablePtrs[3];
+    /* 0x18 */ char unk_18[0x8];
+} JpegTaskData; // size = 0x20
+
+typedef struct {
+    /* 0x000 */ JpegTaskData taskData;
+    /* 0x020 */ char yieldData[0x200];
+    /* 0x220 */ JpegQuantizationTable qTables[3];
+    /* 0x3A0 */ u8 codesLenghts[0x110];
+    /* 0x4B0 */ u16 codes[0x108];
+    /* 0x6C0 */ u16 unk_6C0[4][0x180];
+} JPEGWork; // size = 0x12C0
```

Comment:
I saw that JPEGWork is the original name but for consistency with the other structs I suggest changing it to JpegWork.
```suggestion
} JpegWork; // size = 0x12C0
```